content/posts/2015-09-02-heroku-build-static.md

```diff
@@ -0,0 +1,119 @@
+title: Node.js static file build steps in Python Heroku apps
+date: 2015-09-02
+category: posts
+tags: [heroku, python, node]
+---
+
+Python webapps are pretty cool. Most frontend tools end up getting written in
+Node.js. Things like Less for nicer style sheets, Babel to write modern JS
+today, or NPM to manage front-end dependencies. The result is a lot of webapps
+are polyglots that require more than just Python or Node.
```

Comment:
The start is a bit weak here. I think the things you're trying to say here will benefit from a more personal tone:

> I write a lot of Python webapps, but most frontend tools are written in Node.JS. LESS gives me nicer style sheets, Babel lets me write next-generation JavaScript, and NPM helps manage my front-end dependencies. As a result, most of my webapps are multi-language projects.